IGC/VectorCompiler/lib/GenXCodeGen/GenXLiveness.cpp

```diff
@@ -587,7 +587,7 @@ void GenXLiveness::replaceValue(SimpleValue OldVal, SimpleValue NewVal)
  */
 LiveRange *GenXLiveness::getOrCreateLiveRange(SimpleValue V)
 {
-  auto [i, isInserted] = LiveRangeMap.insert(LiveRangeMap_t::value_type(V, 0));
+  auto [i, isInserted] = LiveRangeMap.emplace(V, nullptr));
```

Comment:
/builddir/build/BUILD/intel-graphics-compiler-igc-1.0.9933/IGC/VectorCompiler/lib/GenXCodeGen/GenXLiveness.cpp:590:58: error: extraneous ')' before ';'

There is extra ) at the end. Works flawlessly otherwise, thanks!

Reply:
Oops! thanks, I'll force-push. I need to update the commit msg slightly as well, turns out there is a libstdc++ bug making this fail. But this change is an improvement anyway.